quickstack/manifests/backups.pp

```diff
@@ -101,20 +99,26 @@
     }
 
     file { 'logrotate':
-        ensure => $ens_file,
-        path  => '/etc/logrotate.d/backups',
-        owner => 'root',
-        group => 'root',
-        content => "${log_file} {\n\tsize 100K\n\tmissingok\n\trotate 6\n\tcompress\n}\n",
+        ensure   => $ens_file,
+        path     => '/etc/logrotate.d/backups',
+        replace  => true,
+        owner    => 'root',
+        group    => 'root',
+        content  => "${log_file} {\n\tsize 100K\n\tmissingok\n\trotate 6\n\tcompress\n}\n",
     } 
 
+    file { 'backup_cron':
+    	ensure     => $ens_file,
+        path       => '/etc/cron.d/backups',
+        replace    => true,
+        owner      => 'root',
+        group      => 'root',
+        mode       => '600',
+        content    => "#This file is managed by Puppet\n#run a daily backup script\nMAILTO=${cron_email}\n${cron_min} ${cron_hour} * * * ${script_dest} -d ${backups_dir} -k ${keep_days} ${v_flag} 2>&1 >>${log_file} | tee -a ${log_file}\n", 
+    }
+
     cron { 'backup_cron':
-      ensure      => $ens_cron, 
-      command     => "${script_dest} -d ${backups_dir} -k ${keep_days} ${v_flag} 2>&1 >>${log_file} | tee -a ${log_file}", 
-      user        => 'root',
-      environment => "MAILTO=${cron_email}",
-      hour        => $cron_hour,
-      minute      => $cron_min,
+        ensure   => absent,
```

Comment:
It replaces the crontab entry backup_cron and with the file backup_cron (defined just above) which is a cron.d script.  We need to make sure the crontab entry gets deleted otherwise the scripts will run twice.  

Once all the nodes get this change, we can take out the code removing the crontab entry.